db/migrate/20180906104915_add_userid_to_reports.rb

```diff
@@ -0,0 +1,5 @@
+class AddUseridToReports < ActiveRecord::Migration[5.0]
+  def change
+    add_column :reports, :user_id, :integer
```

Comment:
In case you decide to update the models relationship, this migration could be removed.